pyproject.toml

```diff
@@ -60,7 +58,6 @@ tomlkit = ">=0.11.4,<1.0.0"
 trove-classifiers = ">=2022.5.19"
 virtualenv = "^20.22.0"
 xattr = { version = "^0.10.0", markers = "sys_platform == 'darwin'" }
-urllib3 = "^1.26.0"
```

Comment:
poetry still has a direct dependency on `urllib3` - https://github.com/python-poetry/poetry/blob/50e61e696bb0ae61c148d570f1c204a667464f1c/src/poetry/publishing/uploader.py#L20 - so this should not be removed.

Reply:
Seemed really minor for me to be kept but I understand the reasoning, better to be safe than sorry!

Reply:
Would you prefer to relax the constraint or pin it to `^2.0.2` to enforce the usage of a more recent version? I'd prefer the second but I also understand the Poetry direct dependency on urllib3 works ok with <2 too..

Reply:
so far as I can see that import is only used to set up a `Retry`, which is configured to retry only for GETs; and the uploader never does a GET, it only POSTs.

So if that's right then that little pile of code could be deleted anyway.   But double-check, I only skimmed it..!

Reply:
I think you are totally right. Wouldn't it be better to then fix it to "POST" if the original code was to retry uploads?

Reply:
meh, if the code has been there for this long without anyone feeling the need to have working retries - I'd just delete it.

Reply:
I was just writing that 😄 Makes sense